baldrick/plugins/tests/test_github_pull_requests.py

```diff
@@ -21,7 +21,7 @@ def setup_module(module):
 
 
 def teardown_module(module):
-    PULL_REQUEST_CHECKS[:] = module.PULL_REQUEST_CHECKS_ORIG[:]
+    PULL_REQUEST_CHECKS = module.PULL_REQUEST_CHECKS_ORIG.copy()
```

Comment:
This won't work as-is unless you make ``PULL_REQUEST_CHECKS`` global, right? I think this should be:

```python
PULL_REQUEST_CHECKS.clear()
PULL_REQUEST_CHECKS.update(PULL_REQUEST_CHECKS_ORIG)
```